examples/time_frequency/plot_epochs_spectra.py

```diff
@@ -33,6 +38,9 @@
 epochs = mne.Epochs(raw, events, event_id, tmin, tmax,
                     proj=True, baseline=(None, 0), preload=True,
                     reject=dict(grad=4000e-13, eog=150e-6))
+# Pull 2**n points to speed up computation
+epochs = EpochsArray(epochs.get_data()[..., :1024], epochs.info,
+                     epochs.events, tmin, epochs.event_id)
```

Comment:
If this is worth making consistent between the two, my intuition is that we should use the behavior of the `crop` function. If a user wants to have a length 1024 array, to me it makes the most sense to do `tmin + 1024. / sfreq`, not do `tmin + 1023. / sfreq` and have to remember that it's inclusive on tmin so it keeps an extra index.